cmd/release-controller/http_helper.go

```diff
@@ -780,9 +780,7 @@ type newestSemVerFromSummaries struct {
 func newNewestSemVerFromSummaries(summaries []UpgradeHistory) newestSemVerFromSummaries {
 	versions := make([]semver.Version, len(summaries))
 	for i, summary := range summaries {
-		if v, err := semver.Parse(summary.From); err != nil {
```

Comment:
Generally this is frowned upon, if a function returns an `err` you check the err, yo udon't assume `v` is uninitialized.  Would suggest removing both these changes.

Reply:
Without updating the logic here, even just this little bit, the sorting doesn't work as I would expect it to.  I have flipped the logic to set the version when these it no error.  Is there something special you would like to do if there is an error?  Log a message or something else?

Reply:
If version is not sem ver, we don't set it.  The new code is correct.